documentation/specs/ProductLayout.md

```diff
@@ -14,8 +14,6 @@
 
 `ProductLayout` will be a compound component consisting of `ProductLayout`, `ProductLayout.Sidebar`, `ProductLayout.Header`, and either `ProductLayout.Content` or `ProductLayout.TabbedContent`.
 
-`ProductLayout` will use "slots" to render subcomponents into the appropriate nested HTML element. See an example `useSlots` reference [implementation](https://github.com/primer/react/blob/main/src/hooks/useSlots.ts#L16). This pattern allows the component surface area to map cleanly to the consumer concerns without having to know about the inner HTML tree.
-
```

Comment:
decided to punt on a more sophisticated slots implementation until we understand more fully how we'll use it in the future. for now, we'll just use props directly vs "transcluding" the children prop.